### PR TITLE
Follow PEP8 in reconst

### PR DESCRIPTION
**Pull request by aaossa**
Using `pycodestyle` fixed every error, except for:

* `csdeconv.py`: The cited lines are docstrings

```
dipy/reconst\csdeconv.py:599:80: E501 line too long (88 > 79 characters)
dipy/reconst\csdeconv.py:601:80: E501 line too long (92 > 79 characters)
```

This is the first PR that replaces #1207

### CODE REVIEW

File: dipy/reconst/csdeconv.py [hunk 1]
```diff
@@ -815,7 +825,11 @@ def auto_response(gtab, data, roi_center=None, roi_radius=10, fa_thr=0.7,
     else:
         ci, cj, ck = roi_center
     w = roi_radius
-    roi = data[int(ci - w): int(ci + w), int(cj - w): int(cj + w), int(ck - w): int(ck + w)]
+    roi = data[
+        int(ci - w): int(ci + w),
+        int(cj - w): int(cj + w),
+        int(ck - w): int(ck + w)
+    ]
```
Comment (arokem) [inline dipy/reconst/csdeconv.py]: Would you mind rewriting this as follows? 

```
data[int(ci - w): int(ci + w),
     int(cj - w): int(cj + w),
     int(ck - w): int(ck + w)]

Comment (aaossa) [inline dipy/reconst/csdeconv.py]: Fixed by 78935bfe97ade3fb76193dc3dfa9e20a5568b577 :ok_hand:

**Comment by aaossa** [comment]
@arokem you kwow what could be causing the build to fail? I'm traying to see if my changes have some problem

**Comment by arokem** [comment]
Even better:

    if no_params > np.sum(~gtab.b0s_mask):

**Comment by aaossa** [comment]
I'll try that right now, hope it finally builds! I think that the problem were caused because we're working with arrays here, I didn't check that, I'm sorry :hear_no_evil:

**Comment by aaossa** [comment]
Thanks for your suggestion @arokem , it fixed the builds :grin: I think now it's ready to be merged

**Comment by arokem** [comment]
LGTM. Anyone else want to take a look?

**Pull request merged**